addons/website/models/website_page.py

```diff
@@ -28,13 +29,19 @@ class WebsitePage(models.Model):
     is_homepage = fields.Boolean(compute='_compute_is_homepage', string='Homepage')
     is_visible = fields.Boolean(compute='_compute_visible', string='Is Visible')
     is_new_page_template = fields.Boolean(string="New Page Template", help='Add this page to the "+New" page templates. It will be added to the "Custom" category.')
+    has_parent_page = fields.Boolean(help="Improve site navigation and hierarchy by incorporating parent pages in the breadcrumb format.")
+    parent_page_id = fields.Many2one('website.page', string="Parent Page", domain="[('website_id','=?',website_id),('id','!=',id)]", store=True, readonly=False)
```

Comment:
What's the purpose of having two fields if they are independent from each other in this model? I don't think you need `has_parent_page` here, it is only useful in the page properties dialog which has its own transient model

For the `parent_page_id` one, should not you use `parent_id` instead so that `child_of` domain selectors work? + it's actually just better to just name it `parent_id` anyway IMO

Reply:
@qsm-odoo We stored ```has_parent_page``` so that later one we can use that field in transient model, if transient model is opened later on then we have to get last stored value from somewhere, for second point I agree, we will change field name to ```parent_id```

Reply:
But can't the transient model guess that from the parent_id field? :thinking: